src/scss/overrides/_type.scss

```diff
@@ -1,10 +1,13 @@
 h1 {
   margin: 20px 0;
   font-size: 3rem;
-  line-height: 1.2;
+  line-height: 1;
   letter-spacing: -0.1rem;
   font-weight: 400;
   color: $charcoal;
+  @media screen and (min-width: 544px) {
```

Comment:
Can we leverage Bootstrap's predefined breakpoints instead of hardcoding our own?

See: http://v4-alpha.getbootstrap.com/layout/overview/#responsive-breakpoints